mypy/config_parser.py

```diff
@@ -60,14 +60,31 @@ def parse_version(v: str | float) -> tuple[int, int]:
     return major, minor
 
 
-def try_split(v: str | Sequence[str], split_regex: str = "[,]") -> list[str]:
-    """Split and trim a str or list of str into a list of str"""
+def try_split(v: str | Sequence[str] | Any, split_regex: str = "[,]") -> list[str]:
```

Comment:
Ah, this should be object, not Any, shouldn't it?

And while I'm at it, I can simplify the split regex

```suggestion
def try_split(v: str | Sequence[str] | object, split_regex: str = ",") -> list[str]:
```